packages/blanket/stories/Blanket.stories.tsx

```diff
@@ -43,5 +37,8 @@ const BasicBlanket = () => {
 
 storiesOf('Blanket', module)
   .add('Basic', () => (
-    <BasicBlanket />
```

Comment:
not enough example with default theme

Reply:
What do you mean?

Should it in finished variant look like this?
- Default
- With isTinted
- With onBlanketClicked
- With canClickThrough
- Themed

Reply:
I mean, in the basic story, a custom theme is passed, although just need to display the component without passing parameters

> Should it in finished variant look like this?
Default
With isTinted
With onBlanketClicked
With canClickThrough
Themed

look good but up to you

Reply:
I pass emptyTheme to prevent warning from withTheme.